api/v1beta1/glance_webhook.go

```diff
@@ -59,16 +59,12 @@ func (r *Glance) Default() {
 
 // Default - set defaults for this Glance spec
 func (spec *GlanceSpec) Default() {
-	if spec.ContainerImage == "" {
+	if len(spec.ContainerImage) == 0 {
 		spec.ContainerImage = glanceDefaults.ContainerImageURL
 	}
```

Comment:
Did you notice the old form of the if-statement failing to work properly in certain cases?

Reply:
I'm fine with as you have it now.  Was just curious if you had seen something.